### PR TITLE
Create psammead-synopsis

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #NUMBER

**Overall change:**
Creates a Synopsis component for Articles

**Code changes:**

- Creates a new package called psammead-synopsis

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-synopsis/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+
+const Synopsis = styled.p`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansBold(service)}
+  color: ${C_METAL};
+  padding-bottom: 2 * ${GEL_SPACING_DBL};
```
Comment (sareh) [inline packages/components/psammead-synopsis/src/index.jsx]: You can import and use `GEL_SPACING_QUAD` instead. 
You should likely reset paragraph padding too on the other SIDES, to override default paragraph spacing. e.g.
`padding: 0 0 ${GEL_SPACING_QUAD} 0;`

File: packages/components/psammead-synopsis/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { shape, string } from 'prop-types';
+import { C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+
+const Synopsis = styled.p`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansBold(service)}
```
Comment (sareh) [inline packages/components/psammead-synopsis/src/index.jsx]:
```suggestion
  ${({ service }) => getSansBold(service)};
```

**Review by sareh — COMMENTED** [review]
Looks great! Great to see a PR from you here, too. :)
I've added a couple of syntax suggestions below.

**Review by sareh — COMMENTED** [review]
This looks good as an alpha component, in terms of being functional and rendering correctly.  I'd like to see a screenshot of designs or requirements listed, to ensure it's matching UX requirements. 

Synopsis story for latin text
Screenshot news latin story
Synopsis story with arabic text
Looking at the story example with an inline link, there may be an accessibility issue with the text intersecting the link underline.  This can be looked at in a UX review of the component. 
Screenshot arabic story

**Comment by jimjohnsonrollings** [comment]
I'm going to close this rather than keep it hanging around - it's not going to be a priority for a while.

**Pull request closed without merging**